components/index.ts

```diff
@@ -1,96 +1,393 @@
+export type {  } from './form/ImageList.js';
+import ImageList from './form/ImageList.js';
+
+//input
+export type { InputProps } from './form/Input.js';
+import Input from './form/Input.js';
+
+//markdown editor
+export type { MarkdownEditorProps } from './form/MarkdownEditor.js';
+import MarkdownEditor from './form/MarkdownEditor.js';
+
+//mask input
+export type { MaskInputProps } from './form/MaskInput.js';
+import MaskInput from './form/MaskInput.js';
+
+//metadata
+export type {  } from './form/Metadata.js';
+import Metadata from './form/Metadata.js';
+
+//number input
+export type { NumberInputProps } from './form/NumberInput.js';
+import NumberInput from './form/NumberInput.js';
+
+//password input
+export type { PasswordInputProps } from './form/PasswordInput.js';
+import PasswordInput from './form/PasswordInput.js';
+
+//radio
+export type { RadioProps } from './form/Radio.js';
+import Radio from './form/Radio.js';
+
+//rating
+export type {  } from './form/Rating.js';
+import Rating from './form/Rating.js';
+
+//select
+export type { SelectProps } from './form/Select.js';
+import Select from './form/Select.js';
+
+//slider
+export type { SliderProps } from './form/Slider.js';
+import Slider from './form/Slider.js';
+
+//slug input
+export type { SlugProps } from './form/SlugInput.js';
+import Slug from './form/SlugInput.js';
+
+//suggest input
+export type { SuggestInputProps } from './form/SuggestInput.js';
+import SuggestInput from './form/SuggestInput.js';
+
+//switch
+export type { SwitchProps } from './form/Switch.js';
+import Switch from './form/Switch.js';
+
+//tag list
+export type { TaglistProps } from './form/TagList.js';
+import Taglist from './form/TagList.js';
+
+//textarea
+export type { TextareaProps } from './form/Textarea.js';
+import Textarea from './form/Textarea.js';
+
+//text editor
+export type { TextEditorProps } from './form/TextEditor.js';
+import TextEditor from './form/TextEditor.js';
+
+//text list
+export type { TextlistProps } from './form/TextList.js';
+import Textlist from './form/TextList.js';
+
+//time input
+export type { TimeInputProps } from './form/TimeInput.js';
+import TimeInput from './form/TimeInput.js';
+
+export {
+  Checkbox,
+  CodeEditor,
+  ColorInput,
+  CountrySelect,
+  CurrencySelect,
+  DateInput,
+  DatetimeInput,
+  FieldControl,
   Fieldset,
-  setColorClass,
-  setBackgroundColorClass,
-  setBorderColorClass,
-  setTextColorClass,
-  setTextAlignClass,
-  setDisplayClass,
-  setRadiusClass,
-  setTextSizeClass
+  FileInput,
+  FileList,
+  ImageInput,
+  ImageList,
+  Input,
+  MarkdownEditor,
+  MaskInput,
+  Metadata,
+  NumberInput,
+  PasswordInput,
+  Radio,
+  Rating,
+  Select,
+  Slider,
+  Slug,
+  SuggestInput,
+  Switch,
+  Taglist,
+  Textarea,
+  TextEditor,
+  Textlist,
+  TimeInput
+};
+
+//--------------------------------------------------------------------//
+// Tool
+
+//--------------------------------------------------------------------//
+// View
```

Comment:
This entire section is incorrectly done. The `export type` lines are correct but you should have done as we discussed before.

```
import * as View from './view/index.js';
```

then export it. make sure there is only one `export {...};`  in the file.